packages/friends/yml/en/george-whitehead.yml

```diff
@@ -9,14 +9,14 @@ residences:
   - city: Sedbergh
     region: England
 description:
-  George Whitehead (1636-1723) was one of the most well-known and influential ministers
-  among the early Society of Friends. He was convinced of the Truth when only 17 years
-  old, and through faithfulness grew quickly in the life and power of the Christ. In his
```

Comment:
The only things I changed here was `among` to `in`, where it said `among the early Society`, and then removed the `the` in `life and power of the Christ.`